Solutions/Silverfort/SolutionMetadata.json

```diff
@@ -0,0 +1,15 @@
+{
+	"publisherId": "silverfort5120441444412",
+	"offerId": "silverfort",
```

Comment:
Please include sentinel keyword in offer id 

<img width="392" alt="image" src="https://github.com/user-attachments/assets/0c62214a-dab1-4f25-9122-c09837a5adb5">

Reply:
Resolved